games-util/lutris/lutris-0.4.23.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{4,5,6} )
+PYTHON_REQ_USE="sqlite,threads"
+
+inherit distutils-r1 gnome2-utils python-r1
+
+DESCRIPTION="Lutris is an open source gaming platform for GNU/Linux."
+HOMEPAGE="https://lutris.net/"
+
+if [[ "${PV}" == "9999" ]] ; then
+	EGIT_REPO_URI="https://github.com/lutris/${PN}.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/lutris/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="GPL-3"
+SLOT="0"
+
+RDEPEND="
+	dev-python/dbus-python[${PYTHON_USEDEP}]
+	dev-python/pygobject:3[${PYTHON_USEDEP}]
+	dev-python/python-evdev[${PYTHON_USEDEP}]
+	dev-python/pyyaml[${PYTHON_USEDEP}]
+	net-libs/libsoup
+	x11-apps/xrandr
+	x11-apps/xgamma"
```

Comment:
You could sort this list alphabetically too, since you most likely have to rebase anyway.